README.md

```diff
@@ -158,44 +158,32 @@ Offered Cms Blocks and Elements are designed to work on pages of type `LandingPa
 There is a type `CategoryPage` but the builtin validation will not allow saving that prepared page, unless it contains at least one default Product Listing Block.
 The block `FACTFinder Web Components Listing` is unfortunately not taken into account.
 
-## Exporting Feed
```

Comment:
This suppose to not be changed to CMS feed because standard Feed still exist.
I'd rather name section `Exporting Feeds`
and add subsections for specic entities
* products
* cms pages

Reply:
Yes, sounds legit, gonna change it